x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/X-BigArrayBlock.java.st

```diff
@@ -8,14 +8,14 @@
 package org.elasticsearch.compute.data;
 
 import org.apache.lucene.util.RamUsageEstimator;
-import org.elasticsearch.common.util.$if(boolean)$Bit$else$$Type$$endif$Array;
+import org.elasticsearch.common.util.$Array$;
```

Comment:
:+1:

Reply:
I was expecting someone to ask me to remove this, so I'm glad you like it. When I had PointBlock, this was very necessary, but even without PointBlock I think this looks a lot nicer, so pleased to keep it.